### DISCUSSION TITLE
blog/logic-of-psychological-counseling

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#16] [Announcements]
# blog/logic-of-psychological-counseling

把自己面临的思想问题看成心理问题，是一种偷懒行为。

https://rercel.com/blog/logic-of-psychological-counseling

**Comment by ghost** [comment]
看到关于心理咨询的讨论非常有趣，心理咨询本质上是辅助人通过“内在自省”改善自我认知能力（大部分精神科/需要心理咨询的人都有这方面的缺陷），认知偏差特别是归因谬误（误以为是自身问题），被困在错误认知中又不断挣扎，其实是缺失“自我预期的管理”和“自我叙事的合法性和完备性”。但市场化的心理咨询就有可能刻意使得客户依赖上这样的服务，而不是通过认知行为疗法CBT或辩证行为疗法DBT去解决。

知识付费我也曾思考过，为何存在“知识焦虑”和“知识付费”，理论上知识在这个时代是最不缺乏的，就像是散发某种错失恐惧症FOMO，特别是知乎的风气：毫无掩饰的精致利己主义，又透露出“赚快钱”、消费主义等主流格调。

我的想法是：现在正是信息过多，若不多加筛选，留给人独立思考的时间很少，依赖现代化的制度来解决私人问题。
